lib/algoliax/resources/object.ex

```diff
@@ -1,11 +1,13 @@
 defmodule Algoliax.Resources.Object do
   @moduledoc false
 
-  import Algoliax.Utils, only: [index_name: 2, object_id_attribute: 1]
+  import Algoliax.Utils, only: [index_name: 2, object_id_attribute: 1, render_result: 1]
   import Algoliax.Client, only: [request: 1]
 
   alias Algoliax.TemporaryIndexer
 
+  # TODO: mutli-query call
+  # TODO: retour save query
```

Comment:
Can we register this in JIRA with a ticket for instance to keep track of it? `TODO(WTTJ2-XXXX)` 

Do you think we would do it in a short term or not required right now?

Reply:
For the `mutli-query call` I think it's alright for now but would be a nice addition, I'll create an issue
The second item has beed fixed :)